engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -384,6 +381,22 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     ()
   }
 
+  private def invalidateCacheHitAndTransition(cacheId: CallCachingEntryId, data: ResponsePendingData, reason: Throwable) = {
+    val invalidationRequired = effectiveCallCachingMode match {
+      case CallCachingOff =>
+        log.error("Should not be calling invalidateCacheHit if call caching is off!")
```

Comment:
I probably wouldn't bother with the exception since it doesn't look like the stack trace would be informative.  And yeah, this will never happen anyway.  😄